community/modules/scheduler/gke-cluster/README.md

```diff
@@ -155,7 +155,7 @@ limitations under the License.
 
 | Name | Description |
 |------|-------------|
-| <a name="output_cluster_id"></a> [cluster\_id](#output\_cluster\_id) | An identifier for the resource with format projects/<project\_id>/locations/<region>/clusters/<name>. |
+| <a name="output_cluster_id"></a> [cluster\_id](#output\_cluster\_id) | An identifier for the resource with format projects/{{project\_id}}/locations/{{region}}/clusters/{{name}}. |
```

Comment:
This portion of the file is auto-generated. I am surprised that it would pass pre-commit if the description in the outputs.tf file was not also updated.

Reply:
Fixed. Descriptions changed and pre-commit run.
I had missed running pre-commit manually for this commit. When pre-commit run automatically on 'git commit', it skips some steps.

Reply:
If you change the terraform then it should update the readme, but if you update the readme then it probably does not run the precommit.